Conceptpower+Spring/src/main/java/edu/asu/conceptpower/core/Comment.java

```diff
@@ -0,0 +1,33 @@
+package edu.asu.conceptpower.core;
+
+import java.time.LocalDateTime;
+
+public class Comment{
+	private String comment;
+	private String createdBy;
+	private LocalDateTime createdAt;
```

Comment:
LocalDateTime doesn't have any timezone information attached to it, which doesn't work in most applications (including this). Use OffsetDateTime instead.